core/trino-main/src/main/java/io/trino/metadata/FunctionMetadata.java

```diff
@@ -30,6 +30,7 @@
     private final String description;
     private final FunctionKind kind;
     private final boolean deprecated;
+    private final String resolvedName;
```

Comment:
A better name for this would be "canonicalName"

Reply:
@kasiafi Thanks for your comments, that helps a lot. 🙇